darwin/importer/importer.py

```diff
@@ -267,6 +275,260 @@ def _resolve_annotation_classes(
+            or annotation.annotation_class.annotation_type
+        )
+        annotation_class_id = int(annotation_class_ids_map[(annotation_name, annotation_type)])
+        annotation_id_property_map[annotation_class_id] = defaultdict(list)
+
+        # raise error if annotation class not present in metadata
+        if (annotation_name, annotation_type) not in metadata_classes_lookup:
+            raise ValueError(
+                f"Annotation: '{annotation_name}' not found in {metadata_path}"
+            )
+
+        # loop on annotation properties and check if they exist in metadata & team
+        for a_prop in annotation.properties or []:
+            a_prop: SelectedProperty
+
+            # raise error if annotation-property not present in metadata
+            if (annotation_name, a_prop.name) not in metadata_cls_prop_lookup:
+                raise ValueError(
+                    f"Annotation: '{annotation_name}' -> Property '{a_prop.name}' not found in {metadata_path}"
+                )
+
+            # get metadata property
+            m_prop: Property = metadata_cls_prop_lookup[(annotation_name, a_prop.name)]
+
+            # get metadata property type
+            m_prop_type: PropertyType = m_prop.type
+
+            # get metadata property options
+            m_prop_options: List[Dict[str, str]] = m_prop.options or []
+
+            # check if property value is missing for a property that requires a value
+            if m_prop.required and not a_prop.value:
+                raise ValueError(
+                    f"Annotation: '{annotation_name}' -> Property '{a_prop.name}' requires a value!"
+                )
+
+            # check if property and annotation class exists in team
+            if (a_prop.name, annotation_class_id) \
+                not in team_properties_annotation_lookup:
+
+                # check if fullproperty exists in create_properties
+                for full_property in create_properties:
+                    if full_property.name == a_prop.name and \
+                        full_property.annotation_class_id == annotation_class_id:
+                        # make sure property_values is not None
+                        if full_property.property_values is None:
+                            full_property.property_values = []
+
+                        property_values = full_property.property_values
+                        # find property value in m_prop (.v7/metadata.json) options
+                        for m_prop_option in m_prop_options:
+                            if m_prop_option.get("value") == a_prop.value:
+                                # update property_values with new value
+                                full_property.property_values.append(
+                                    PropertyValue(
+                                        position=m_prop_option.get("position"),  # type: ignore
+                                        type=m_prop_option.get("type"),  # type: ignore
```

Comment:
We don't have this field for property value in the manifest anymore. Please make fresh export and check. We don't need to pass it in the API either.